API/ev3_lcd.c

```diff
@@ -1688,30 +1672,52 @@ char RectOutEx(int x, int y, int width, int height, unsigned long options)
 	return 0;
 }
 
-char TextOutEx(int x, int y, char* str, unsigned long options)
+char EllipseOutEx(int x, int y, uint8_t radiusX, uint8_t radiusY, unsigned long options)
 {
 	if (!LcdInitialized())
 		return 1;
-//  uint8_t pixelMode, fillMode;
-//  if (CmdResolveDrawingMode(options, &pixelMode, &fillMode))
-//    CmdDrawRect(x, y, width, height, pixelMode);
+	uint8_t pixelMode, fillMode;
+	if (CmdResolveDrawingMode(options, &pixelMode, &fillMode))
+		CmdDrawEllipse(x, y, radiusX, radiusY, pixelMode, fillMode);
 	return 0;
 }
 
-char NumOutEx(int x, int y, int value, unsigned long options)
+
+/**********************************************************************
+ * Unused / currently useless functions
+ */
+#if 0

```

Comment:
Oh, I would just delete them. It's unlikely they will be ever implemented.

Reply:
done